integrations/amplitude/test/index.test.js

```diff
@@ -27,7 +27,8 @@ describe('Amplitude', function() {
     traitsToIncrement: [],
     traitsToSetOnce: [],
     preferAnonymousIdForDeviceId: true,
-    unsetParamsReferrerOnNewSession: false
+    unsetParamsReferrerOnNewSession: false,
+    versionName: '4.6.0'
```

Comment:
Since we want to keep backwards compatibility, let's add a test that initializes Amplitude without `versionName` (ie without that setting), and check that `config.versionName === undefined` (or whatever value Amplitude assigns by default.

Reply:
In that case it pick the default version that sdk uri has **5.2.2** 
var src = 'https://cdn.amplitude.com/libs/amplitude-5.2.2-min.gz.js';

Reply:
Uhm, I think here `versionName` doesn't refer to the Amplitude SDK version, but what name the customer assigned to their webpage code. I would imagine by default `config.versionName` would be `null` or `undefined`.